src/main/resources/css/common.css

```diff
@@ -177,7 +202,7 @@ and (min-device-width: 320px) {
         width: 70%;
     }
     .searchbar {
-        min-width: 312px;
+        min-width: 290px;
```

Comment:
where did we get this width for the search bar from?

Reply:
This may not be the best solution as I was trying to make it look good for iphone5 version. If there is a better way then I'm happy to go with.

Reply:
Oh ok... we could change it from `min-width` to `max-width: 312px` and then set `width: 80%`, that would probably be a better way.

Reply:
Fixed